docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
"I have come to recognize that integration testing is probably a better approach which is why redux-saga-test-plan now offers it with a simple, declarative API." I can't agree more. In fact, at my work we've come to the firm conclusion that unit testing sagas offers no benefit. The tests are tautological. And aside from that, you have to put in the substantial time investment of writing them and maintaining them (anytime any effects in the tested sagas change, you have to update its unit tests). We're going to delete our saga unit tests and search for something closer to an integration testing solution.